cfme/configure/configuration/server_settings.py

```diff
@@ -328,6 +328,13 @@ def update_vmware_console(self, updates, reset=False):
                         product_name=self.appliance.product_name,
                         version=self.appliance.version
                     )
+            if value == 'VMware WebMKS':
```

Comment:
**Required:** Please move this under  if name == 'console_type', to check console type only if this is console